DependencyInjection/DoctrinePHPCRExtension.php

```diff
@@ -210,6 +210,14 @@ private function loadJackalopeSession(array $session, ContainerBuilder $containe
 
         // pipe additional parameters unchanged to jackalope
         $backendParameters += $session['backend']['parameters'];
+        if (array_key_exists('curl_options', $session['backend']) && count($session['backend']['curl_options'])) {
+            $curlOptions = array();
+            foreach ($session['backend']['curl_options'] as $option => $value) {
+                $curlOptions[constant('CURLOPT_'.$option)] = $value;
```

Comment:
that would be possible! it would also be possible to uppercase the `$option` that should be enough.